types.go

```diff
@@ -342,6 +343,34 @@ func (s *RemoteSensor) Temperature() (float64, error) {
 	return 0.0, fmt.Errorf("remote sensor %v does not have a temperature capability", s.Name)
 }
 
+// Humidity gets the humidity for the sensor if it exists.
+func (s *RemoteSensor) Humidity() (int, error) {
+	if s != nil && len(s.Capability) > 0 {
+		for _, c := range s.Capability {
+			if c.Type == CapabilityTypeHumidity {
+				v, err := strconv.ParseInt(c.Value, 0, 0)
+				if err != nil {
+					return 0.0, err
```

Comment:
Should just be `0` here to reduce confusion, since this function returns `int`.

```suggestion
					return 0, err
```

Reply:
Good spot -- thanks! GitHub's new (to me) "Commit Suggestion" looks like it'll create a separate commit, which seems less than ideal to me, so I fixed this locally and re-pushed.  PTAL!

re: promobee -- it's working great for me.  I've already extended it to plot Humidity, HVAC Status (off/heat/cool), HVAC Operation (cooling, heating, fan status), as well as occupancy status.

I'll send some PRs once I've got this merged (and worked out how to update the vendored egobee in promobee; I'm relatively new to golang vendoring, so any pointers there would be appreciated!)

Reply:
Glad to hear it! You've motivated me to get mine running again. 😃